src/UI/PlayerSetupColorFactionController.java

```diff
@@ -63,12 +65,13 @@ public boolean handleInput(InputAction action)
             myPlayerInfo.coList.get(i).color   = colorPicks  [i];
             myPlayerInfo.coList.get(i).faction = factionPicks[i];
           }
+          myPlayerInfo.flipUnits = flipUnits;
```

Comment:
Oh, snap. It is not clear at all from the UI that this is a change that will carry into the game.
I sorta like having it be based on start position (because that's simple and sensible and transparent), but uh. I guess.
Maybe use the old way as the default (if you aren't already)?
Also maybe improve the tooltip to clarify this is an in-game change, not just so you can see the sprite both ways.

Reply:
I feel like the facing being different in the main player setup screen is a solid hint, but there's probably more I can do.

Reply:
It is, but it's a hint you only see after you've made the change. The wording tweak feels like enough to make it more obvious.